src/AMD_stability/MMR3pl_overlap.jl

```diff
@@ -0,0 +1,104 @@
+#=
+Date: 8 October 2020
+
+Author: A. C. Petit
+
+Functions giving spacing limits for multiplanetary systems based on the overlap of zeroth order 3 planets systems.
+Based on A. C. Petit, G. Pichierri, M.B. Davies, A. Johansen 2020 (https://ui.adsabs.harvard.edu/abs/2020A%26A...641A.176P/abstract)
+
+=#
+
+"""
+    MMRoverlapcriterion(ν12::Real,ν23::Real,ϵ1::Real,ϵ2::Real,ϵ3::Real,fudge::Real=1)
+
+Tests if the three planet MMR overlap and returns a symbol (`:Stable` or `:MMR3ploverlap`) as well as the ratio of `δ/δov` of the spacing over the overlap spacing (larger than 1 means stable).
+
+νij is the period ratio between planet i and j, ϵi is the planet i-to-star mass ratio, `fudge` corresponds to a factor that can be added to increase the amount of resonances, simulating the effect of additionnal planets in the system.
+"""
+function MMRoverlapcriterion(ν12::Real,ν23::Real,ϵ1::Real,ϵ2::Real,ϵ3::Real,fudge::Real=1)
+    α12 = ν12^(2/3)
```

Comment:
When raising to fractional powers, writting it like `x^(2//3)` might allow the compiler to optimize by doing a square followed by a cube root, rather than having to deal with an arbitrary floating point power.

Reply:
I actually tested that and after benchmarking, it turns out the the rational version is much slower than the floating point.

```
ffloat(x) = x^(2/3)
frat(x) = x^(2//3) 
```

> julia> @btime ffloat(1.3)
>   1.421 ns (0 allocations: 0 bytes)
> 1.1911384251964328
> 
> julia> @btime frat(1.3)
>   27.681 ns (0 allocations: 0 bytes)
> 
>  1.1911384251964328